packages/webapp/pages/squads/create.tsx

```diff
@@ -121,6 +121,7 @@ function CreatePost(): ReactElement {
     onSuccess: (newSquad) => {
       const form = formToJson<CreatePostProps>(formRef.current);
 
+      setSelected(selected + 1);
```

Comment:
Moving the "create new squad" option to the bottom creates some jank in the UI if the navigation is slow after creating the post:

https://github.com/user-attachments/assets/a6895a54-bcde-4c54-b99d-e9e3b8e10530

It happens because the list of squads gets updated before the navigation happens.

Updating the selected index was the quickest and easiest fix. It works because the update of the list of squads and setSelected are batched together.